rusk/src/bin/main.rs

```diff
@@ -73,8 +76,9 @@ async fn main() {
                 .default_value("info")
                 .help("Output log level")
                 .takes_value(true),
-        )
-        .get_matches();
+        );
+    let app = self::network_config(app);
```

Comment:
`self` is a bit confusing here.

```suggestion
    let app = network_config(app);
```